Document Master URL format in high availability set up

MirrorZ opened the pull request:
## What changes were proposed in this pull request?

Add documentation for adding master url in multi host, port format for standalone cluster with high availability with zookeeper.
Referring documentation [Standby Masters with ZooKeeper](http://spark.apache.org/docs/latest/spark-standalone.html#standby-masters-with-zookeeper)

## How was this patch tested?

Documenting the functionality already present.

MirrorZ committed:
Document Master URL format in high availability set up

Add documentation for adding master url in multi host, port format for standalone cluster with high availability with zookeeper.
Referring documentation [Standby Masters with ZooKeeper](http://spark.apache.org/docs/latest/spark-standalone.html#standby-masters-with-zookeeper)

AmplabJenkins commented:
Can one of the admins verify this patch?

srowen commented on docs/submitting-applications.md:
```diff
@@ -143,6 +143,9 @@ The master URL passed to Spark can be in one of the following formats:
 <tr><td> <code>spark://HOST:PORT</code> </td><td> Connect to the given <a href="spark-standalone.html">Spark standalone
         cluster</a> master. The port must be whichever one your master is configured to use, which is 7077 by default.
 </td></tr>
+<tr><td> <code>spark://HOST1:PORT1,HOST2:PORT2</code> </td><td> Connect to the given <a href="spark-standalone.html#standby-masters-with-zookeeper">Spark standalone
```
zookeeper -> Zookeeper
Isn't this already documented then? I get that sometimes it's useful o doc something two places but I wonder if this is just redundant.

MirrorZ replied:
I came across a situation where the developers using the spark driver believed that it was not possible to accept hosts in the mentioned format because it wasn't mentioned in the docs, so I found it useful to add the detail and submit a PR.
Cheers!

MirrorZ replied:
I came across a situation where the developers using the spark driver believed that it was not possible to accept hosts in the mentioned format because it wasn't mentioned in the docs, so I found it useful to add the detail and submit a PR.
Cheers!

MirrorZ committed:
zookeeper -> Zookeeper

Update master url description for high availability master set up with Zookeeper, changing zookeeper -> Zookeeper.

SparkQA commented:
**[Test build #3657 has started](https://amplab.cs.berkeley.edu/jenkins/job/NewSparkPullRequestBuilder/3657/testReport)** for PR 17584 at commit [`67221f7`](https://github.com/apache/spark/commit/67221f7b193f72312abe37eb7b45a54edefea3e8).

SparkQA commented:
**[Test build #3657 has finished](https://amplab.cs.berkeley.edu/jenkins/job/NewSparkPullRequestBuilder/3657/testReport)** for PR 17584 at commit [`67221f7`](https://github.com/apache/spark/commit/67221f7b193f72312abe37eb7b45a54edefea3e8).
 * This patch passes all tests.
 * This patch merges cleanly.
 * This patch adds no public classes.

srowen commented:
merged to master
